src/Playwright.Tests/PageBasicTests.cs

```diff
@@ -204,8 +204,11 @@ public async Task ShouldWorkWithPageClose()
         [Fact(Timeout = TestConstants.DefaultTestTimeout)]
         public async Task ShouldFireLoadWhenExpected()
         {
+            var loadEvent = new TaskCompletionSource();
+            Page.Load += (_, _) => loadEvent.TrySetResult();
```

Comment:
Let's make the tests .NET 3.1 friendly. If we don't want to be 3.1 friendly let's remove the build step.

```suggestion
            Page.Load += (_, _) => loadEvent.TrySetResult(true);
```